src/styles/style.js

```diff
@@ -0,0 +1,15 @@
+import { createGlobalStyle } from "styled-components"
+import InterFont from '../fonts/Inter.var.woff2'
+
+const FontStyles = createGlobalStyle`
+  @font-face {
+    font-family: 'Inter var';
+    font-display: swap;
+    src: url(${InterFont}) format('woff2');
```

Comment:
The trick here is having the `src` `url` being referenced from the imported font rather than a static string.

This way, webpack is able to still resolve the hashed version of the filename.